R/carehomes.R

```diff
@@ -375,6 +378,7 @@ carehomes_parameters <- function(start_date, region,
                                  m_CHW = 4e-6,
                                  m_CHR = 5e-5,
                                  p_NC = 0.01,
+                                 p_NC_weekend = 0.005,
```

Comment:
does this make more sense parametrised relative to p_NC (i.e. a weekend effect of 0.5 for this combination)

Reply:
I guess if you wanted to force `p_NC_weekend` to be less than `p_NC`, but perhaps I'd rather let the data determine that. Also I think it might be better from a fitting POV to fit `p_NC` and `p_NC_weekend`, as these would be less correlated than `p_NC` and some multiplicative weekend effect